dvc_http/__init__.py

```diff
@@ -81,10 +80,10 @@ def _prepare_credentials(self, **config):
         if "ssl_verify" in config:
             connector_kwargs.update(ssl=make_context(config["ssl_verify"]))
 
-        with fsspec_loop():
-            client_kwargs["connector"] = aiohttp.TCPConnector(
-                **connector_kwargs
-            )
+        client_kwargs["connector"] = aiohttp.TCPConnector(
+            loop=self.fs.loop, **connector_kwargs

```

Comment:
Let's move this to `get_client`, otherwise we are creating a circular dependency here (`fs_args` are updated in `__init__()` via `_prepare_credentials()`, and it should not call `fs` which requires `fs_args`).

Also this will break lazy-initialization of filesystem since this function is called in `__init__()`.